src/api/java/appeng/api/definitions/IParts.java

```diff
@@ -107,4 +107,14 @@
 	IItemDefinition storageMonitor();
 
 	IItemDefinition conversionMonitor();
+
+	IItemDefinition fluidTerminal();
+
+	IItemDefinition importBusFluids();
```

Comment:
Still not sure about the naming conventions. Currently it is not really consistent with sometimes using plural, sometimes singular. Also prefix, infix, and postfix is randomly used. Mostly to avoid something like the API name is `importBusFluids`, the class uses `ImportFluidBus` and the texture `import_bus_fluid.png` and the model `fluid_import_bus`. Just not sure about the best one currently, but at first glance something lile `fluidImportBus` makes most sense, also when looking at the terminal with `fluidTerminal`